CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Mailchain
+
+We :heart: your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## Reading
+
+There are 2 important documents to read to understand how we are building Mailchain.
+
+1. We use [tenants](https://github.com/mailchain/mailchain-specification/blob/master/mailchain_tenets.md) to guide the way we think.
```

Comment:
Tenets, not tenants.

guide the way we think >> guide the way we make decisions.